server.js

```diff
@@ -0,0 +1,118 @@
+// var express = require('express');
```

Comment:
Commented out code?

Reply:
I'd forgotten that was there.
Removed now